common/djangoapps/edxmako/shortcuts.py

```diff
@@ -108,7 +109,7 @@ def microsite_footer_context_processor(request):
     """
     return dict(
         [
-            ("IS_REQUEST_IN_MICROSITE", microsite.is_request_in_microsite())
+            ("IS_REQUEST_IN_MICROSITE", is_request_in_themed_site())
```

Comment:
@mattdrayer I do not see any usage of `IS_REQUEST_IN_MICROSITE` in our code, it most probably is being used by other microsite users. renaming it could break their code unless they update it according to the release.
we can add an entry in release notes and rename this field. Let me know what you think ?